requirements/base.txt

```diff
@@ -1,4 +1,4 @@
 Django>=2.0
-tablib[all]>=0.14
+tablib[html,odf,xlsx,yaml]>=0.14
```

Comment:
@claudep this is used on tests, but when installing, setup.py is used. Can you please fix it too?